docs/architecture/adr/0003-switch-test-assertions-from-truth-to-assertk.md

```diff
@@ -0,0 +1,37 @@
+# Switch Test Assertions from Truth to AssertK
+
+## Status
+
+- **Accepted**
+
+## Context
+
+Our project has been using the Truth testing library for writing tests. While Truth has served us well, it is primarily
+designed for Java and lacks some features that make our Kotlin tests more idiomatic and expressive. As our codebase is
+[primarily Kotlin](0001-switch-from-java-to-kotlin.md), we have been looking for a testing library that is more aligned
+with Kotlin's features and idioms.
+
+## Decision
+
+We have decided to use [AssertK](https://github.com/willowtreeapps/assertk) as the default assertions framework for
```

Comment:
The README of the project consistently uses the spelling *assertk* (all lower case). We should respect their stylistic choice.